templates/yamaha_show_ip_route.textfsm

```diff
@@ -0,0 +1,16 @@
+# You need to set "console character ascci".
```

Comment:
Do you mind using looking at `cisco_ios_show_ip_route` and use some of the same names for those? We try and keep values the same between vendors to hopefully allow users to use the same keys for each vendor/command.

Other than that, I think this all looks good.

Reply:
Thank you for your comment. I fixed it.
`additional_info` can contain various information, so I keep the same key name as the result of executing the show command.